### ISSUE TITLE
Backup not starting, stucks at "The creation of a restoring point as been requested and will be initiated soon."

### ISSUE THREAD

**Issue by marcel151** [nextcloud/backup#160]
Unfortunately, the initial Backup is not starting. I also can't get rid of the message above, even after uninstalling and reinstalling the backup app. Issue is described here: https://help.nextcloud.com/t/new-backup-app-setup/129889/3

**Comment by mueslo** [comment]
Stuck for how long? In my case it was like this and then started a few hours later at precisely 5pm local time.

**Comment by marcel151** [comment]
It's now 7 days since I started the initial backup. I don't think that it's started.

**Comment by Gamemap** [comment]
Did you changed the background jobs from `AJAX` to `Cron` ? 
(https://docs.nextcloud.com/server/23/admin_manual/configuration_server/background_jobs_configuration.html#cron)
This solved a similar problem for me.

**Comment by marcel151** [comment]
No, it was still on AJAX and I tried to change it to cron. However, cron don't seem to be working correctly.:
"Last job execution ran 31 minutes ago. Something seems wrong."

I am running Nextcloud as a Docker container.

**Comment by mueslo** [comment]
I also got those error messages, they seem to not directly cause any failures though.

**Comment by Mister-Muffin** [comment]
These errors make the backup process not work entirely.
See previous issue #152

**Comment by arg** [comment]
> These errors make the backup process not work entirely. See previous issue #152

Can also confirm, I'm having the same issue: backup never completes, and I get the error listed in https://github.com/nextcloud/backup/issues/152#issuecomment-1017542650

**Comment by erik78se** [comment]
I can't even get the backup to run at all.

I have just set "cron"

I run 23.0.0.10

backup App version 1.0.4

Is this app maintained?

**Comment by nlovrens** [comment]
> No, it was still on AJAX and I tried to change it to cron. However, cron don't seem to be working correctly: "Last job execution ran 31 minutes ago. Something seems wrong."
> 
> I am running Nextcloud as a Docker container.
> 
> EDIT: Ok it seems that triggering the cron file from the host is working. Not very intuitive, but doesn't have something to do with this app. Nervertheless, I am getting Errors in the log.
> 
> ```
> Error: Undefined variable $user at /var/www/html/custom_apps/backup/lib/Service/ExternalFolderService.php#900
> Error: Undefined variable $user at /var/www/html/custom_apps/backup/lib/Service/ExternalFolderService.php#899
> ```

Hi,

I have the same issue on docker, can you please tell me which cron file has to be triggered?

thanks

**Comment by erik78se** [comment]
The problem seems to be related to the *user* variable not been set. This would happen for example for local storage where authentication is not needed or available?

I'm not an expert so I could be wrong

**Comment by arg** [comment]
Correct, this issue doesn’t seem to be related to cron or any other background jobs implementation. See #152 for details.

Best regards,
Andrii

**Comment by cybergitti** [comment]
Hi, I think this issue is now opened in another id
https://github.com/nextcloud/backup/issues/186
hope, the $user parameter problem will be solved by an expert, because backup is not working at all in my setup.
self hosted ubunty php 8.0, NC23 ...

**Comment by Vrumf0ndel** [comment]
issue with $user parameter is fixed with PR #206.
